pkgs/development/python-modules/tensorflow-datasets/corruptions.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/tensorflow_datasets/image_classification/corruptions.py b/tensorflow_datasets/image_classification/corruptions.py
+index 066c4460..cb9a6667 100644
+--- a/tensorflow_datasets/image_classification/corruptions.py
++++ b/tensorflow_datasets/image_classification/corruptions.py
+@@ -35,7 +35,7 @@ FROST_FILENAMES = []
+ 
+ 
+ def _imagemagick_bin():
+-  return 'imagemagick'  # pylint: disable=unreachable
++  return 'convert'  # pylint: disable=unreachable
```

Comment:
Why is our binary name for imagemagick different?

Reply:
All distributions I've checked (NixOS, Gentoo, Ubuntu) provide `convert` binary for ImageMagick 6.x and `magick` for ImageMagick 7.x (but with fallback link for `convert`). I have no idea where they've got `imagemagick` binary name from. Based on other evidence (see below) it's not clear whether this test worked any time recently?

Reply:
Ah ok, got it! Seems like it's also flagged as unreachable code by pylint anyhow.